tests/unit/test_jinja_templates.py

```diff
@@ -0,0 +1,32 @@
+from itertools import chain
+from pathlib import Path
+
+import pytest
+from jinja2.nodes import Call, FromImport, Name
+
+
+class UnusedJinjaImports(Exception):
+    def __init__(self, unused_imports, file):
+        separator = "\n  • "
+        formatted_unused_imports = separator.join(unused_imports)
+        super().__init__(f"\n  Found unused Jinja imports in {file}:{separator}{formatted_unused_imports}\n\n")
+
+
+root_dir = Path(__file__).parent.parent.parent.resolve()
+files = sorted(root_dir.glob("app/**/templates/**/*.html"))
+
+
+@pytest.mark.parametrize("file", files, ids=[str(path.relative_to(root_dir)) for path in files])
```

Comment:
I believe this will pass (or "skip") if this file gets refactored or moved around an no longer finds any HTML files. Having false confidence in this particular issue probably isn't a huge deal but if it could fail in that instance it would help us catch it (I'm not sure what a nice way to do that is, we might just want to trust the configuration!)